CMakeLists.txt

```
@@ -80,3 +80,5 @@ protocol("unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml" "linux-dmabuf-unst
 
 # Installation
 install(TARGETS hyprlock)
+
+install(FILES "pam/hyprlock" DESTINATION "/etc/pam.d")
```

```suggestion
install(FILES "pam/hyprlock" DESTINATION "${CMAKE_INSTALL_SYSCONFDIR}/pam.d")
```

As per https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.

---

@fufexan using that just makes it install to `/pam.d/hyprlock`
does cmake always need to be called with `-DCMAKE_INSTALL_SYSCONFDIR=/etc`?

---

Oops my bad, forgot we have to also add `include(GNUInstallDirs)` before the`install(...)`.

---

Yes that worked